hivelink-core/src/main/java/org/apache/iceberg/hivelink/core/HiveMetadataPreservingTableOperations.java

```diff
@@ -195,17 +195,36 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
     ReentrantLock tableLevelMutex = commitLockCache.get(fullName, t -> new ReentrantLock(true));
     tableLevelMutex.lock();
     try {
+      LOG.warn(
+          "In thread {}, starting to acquire a lock for table {}",
```

Comment:
Nit: Is it possible to add timestamp before/after acquired the lock?

Reply:
The timestamp should already be included as a prefix in the logging message?

Reply:
Thanks, just checked the container log emitted by Gobblin, looks you are correct